knora/dsplib/models/value.py

```diff
@@ -297,6 +297,8 @@ def __init__(self,
         self._d2 = None if dp[8] is None else int(dp[8].strip('-: '))
         if self._y1 is None:
             raise BaseError("Invalid date format! " + str(value))
+
+        '''
```

Comment:
@subotic @lrosenth Should we improve this code or get rid of it?

If I understand correctly, this code was supposed to check for periods whose start date was after the end date (and then swap them if this is the case). So I do not know how hard it will be to make that work correctly with eras.

Reply:
@subotic @lrosenth What about this logic? Should we fix or simply remove it?

Reply:
I don't understand. I thought you were fixing it?

Reply:
I just commented it out because it does some validation before sending the date to Knora. I think fixing it would take quite some time and I don't think it's worth the effort since Knora would reject an invalid date anyway.

Reply:
So far, the validation logic created a bug ;-)

Reply:
-> malidation